src/EditorFeatures/CSharpTest/IntroduceUsingStatement/IntroduceUsingStatementTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.IntroduceUsingStatement;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.IntroduceUsingStatement
+{
+    [Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceUsingStatement)]
+    public sealed class IntroduceUsingStatementTests : AbstractCSharpCodeActionTest
```

Comment:
are you going to add the feature for VB as well?

Reply:
Probably, once the C# and abstract implementations settle and the tests seem to be mostly written.

Reply:
At this point, I'd like to avoid it unless someone asks for it.